MapboxCoreNavigation/NavigationSettings.swift

```diff
@@ -60,7 +60,7 @@ public class NavigationSettings: NSObject {
         let properties = Mirror(reflecting: self).children
         return properties.filter({ (child) -> Bool in
             if let label = child.label {
-                return label != "properties.storage"
+                return label != "properties.storage" && label != "$__lazy_storage_$_properties"
```

Comment:
I considered changing this check to `!label.starts(with: "$__lazy_storage_$_")`, but the intent of this check was specifically to filter out the `properties` property, not any lazy arbitrary property.